xlive/H2MOD.cpp

```diff
@@ -262,8 +262,8 @@ signed int __cdecl object_new_hook(void *pObject)
 	object_to_variant[result] = variant_index;
 
 	wchar_t DebugText[255] = { 0 };
-	ZeroMemory(DebugText, 255);
-	wsprintf(DebugText, L"AI object_new hook - object_index: %08X - variant_index: %08X - datum: %08X", result,variant_index);
+	ZeroMemory(DebugText, sizeof(DebugText));
```

Comment:
shouldn't this be 255 * sizeof(wchar_t)?

Reply:
yeah i think you're right